posthog/temporal/data_imports/pipelines/pipeline.py

```diff
@@ -63,12 +64,16 @@ def _get_destination(self):
                 "aws_access_key_id": settings.AIRBYTE_BUCKET_KEY,
                 "aws_secret_access_key": settings.AIRBYTE_BUCKET_SECRET,
                 "endpoint_url": settings.OBJECT_STORAGE_ENDPOINT,
+                "region_name": settings.AIRBYTE_BUCKET_REGION,
+                "AWS_ALLOW_HTTP": "true",
+                "AWS_S3_ALLOW_UNSAFE_RENAME": "true",
```

Comment:
Bumping to double check: is this the right place for these credentials or do they need to be set on the worker itself too? I didn't get to test this and I have always had this manually set on the env when running the temporal worker locally

Reply:
These are just for running the local tests. `AWS_ALLOW_HTTP` is needed for delta-rs to talk to local MinIO. They get explicitly passed through to DLT and delta-rs